packages/api/src/jobs/station_sync/updateTimeZoneOffsets.js

```diff
@@ -1,20 +1,66 @@
+/*
+ *  Copyright 2019, 2020, 2021, 2022 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
 const { Client } = require('@googlemaps/google-maps-services-js');
 
 const client = new Client({});
 
 async function mapTimeZoneOffsetsToFarms(knex) {
   const farmsWithLatLng = await knex('farm').select('*').whereNotNull('grid_points');
   for (const farm of farmsWithLatLng) {
-    const timeZone = await client.timezone({
-      params: {
-        location: farm.grid_points,
-        timestamp: new Date(Date.now()),
-        key: process.env.GOOGLE_MAPS_API_KEY,
-      },
-    });
-    await knex('farm')
-      .update('utc_offset', Math.round(timeZone.data.rawOffset / 60))
-      .where('farm_id', farm.farm_id);
+    try {
+      const timeZone = await client.timezone({
+        params: {
+          location: farm.grid_points,
+          timestamp: new Date(Date.now()),
```

Comment:
Suggestion: one new Date at the beginning. I doubt that the differences would matter, but might be better to have the data as of a single point in time.